### DISCUSSION TITLE
AbortController/AbortSignal not available?!

### DISCUSSION THREAD

**Discussion by KG3RK3N** [NativeScript/NativeScript#9867] [Q&A] [answered]
Hello,

I would work with abort controllers & signals for cancellation of complex tasks. I found on the [NS 8.1 Announcement](https://blog.nativescript.org/nativescript-8-1-announcement/index.html#features) that the AbortController polyfill is one of the new features of that version. But after a issue on usage and some research I found a [PR](https://github.com/NativeScript/NativeScript/commit/c68d88d1d3b8142507d42423bad742c8b2999a34) by @NathanWalker where he commented out the registering of that global polyfill.
After I've added the registering again locally the abort controller & signal works perfect. So in my opinion the polyfill should registered again.

Any reasons why that was commented out or plans to add that again?

I use NS 8.1.5

**Comment by farfromrefug** [comment, marked as answer]
@KG3RK3N we decided to not enable it "by default"
You can still enable it with webpack with something like this :
```js
config.plugins.unshift(
        new webpack.ProvidePlugin({
            AbortController: [require.resolve('@nativescript/core/abortcontroller'), 'AbortController'],
            AbortSignal: [require.resolve('@nativescript/core/abortcontroller'), 'AbortSignal'],
        })
    );
```

**Reply by KG3RK3N** [reply]
Thank you @farfromrefug for your fast response.
Ah okay, nice to know. I've tried your sample and it works, great! You made my day👍
